test/test_plugin_baseline.py

```diff
@@ -167,21 +169,12 @@ def test_resource_name_factory_generation(self, run_protoc, oneof):
 
 class TestProtocGapicPluginV2(object):
 
-    @pytest.mark.parametrize('resource', RESOURCE_NAMES_TO_GENERATE)
+    @pytest.mark.parametrize('resource', RESOURCE_NAMES_TO_GENERATE + ["project_name"])
     def test_resource_name_generation(self, run_protoc_v2, resource):
         generated_class = casing_utils.lower_underscore_to_upper_camel(
             resource)
         check_output(generated_class, PROTOC_OUTPUT_DIR, 'java_' + resource)
 
-    @pytest.mark.parametrize('resource', DONT_GENERATE)
```

Comment:
We actually do want to generate `project_name` here, to test the parent of a child_type is generated. Also, project_name here is not marked as a common resource name in the gapic config, so it should be generated.